### ISSUE TITLE
Refit on the whole training set not supported

### ISSUE THREAD

**Issue by ArlindKadra** [automl/Auto-PyTorch#191]
The `fit_pipeline` method does not support refitting on the training set with 0 in the resampling strategy arguments for the validation set.
This should be fixed since it is a common scenario for the library.

**Comment by ArlindKadra** [comment]
Addressed with #190
